File: 2025_02/leetcode2025-02-01.py
from typing import List

class Solution:
    def search(self, nums: List[int], target: int) -> bool:
        return target in nums

if __name__ == "__main__":
    s = Solution()
    nums = [2,5,6,0,0,1,2]
    target = 0
    print(s.search(nums, target))

File: leetcode2025/2025_02/leetcode2025-02-01.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
    bool search(vector<int>& nums, int target) {
        for (int i : nums) if (i == target) return true;
        return false;
    }
};

int main(){
    Solution s;
    vector<int> nums = {2,5,6,0,0,1,2};
    int target = 0;
    cout << s.search(nums, target) << endl;
}
